src/sage/categories/coxeter_groups.py

```diff
@@ -1959,6 +1962,7 @@ def absolute_length(self):
             .. SEEALSO::
 
                 :meth:`absolute_le`
+                :meth:`absolute_chain`
```

Comment:
```suggestion
                :meth:`absolute_le`,
                :meth:`absolute_chain`
```
Since they are not on separate lines in the compiled doc.